src/Tile/Tile.test.js

```diff
@@ -89,6 +89,26 @@ describe('<Tile />', () => {
         </Tile>
     );
 
+    const productTile = (
+        <Tile className='pink' isButton
```

Comment:
What is the `isButton` prop? I'm not seeing it documented anywhere.

Reply:
This was in the previous productTile tests so I assumed it was a prop passed down 😮 - will take it out.